CONTRIBUTING.md

```diff
@@ -318,32 +134,14 @@ uutils: add new utility
 gitignore: add temporary files
 ```
 
-## Code coverage
-
-<!-- spell-checker:ignore (flags) Ccodegen Coverflow Cpanic Zinstrument Zpanic -->
-
-Code coverage report can be generated using [grcov](https://github.com/mozilla/grcov).
+## TODO : Any guidelines for branching, PRs, etc?
```

Comment:
please remove it, we don't do it yet

Reply:
ping ?

Reply:
apologies - have been traveling. will do